src/styles/buy.scss

```diff
@@ -0,0 +1,28 @@
+.buy {
+  display: flex;
+  box-sizing: border-box;
+  width: $defaultWidth;
+  height: $buttonHeight;
+  background-color: $buttonColor;
+  justify-content: center;
+  align-items: center;
+  border-radius: 5px;
+  margin: 16px auto;
+
+  .buyFont {
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    color: #fff;
+  }
+}
+
+.buy:hover {
+  background-color: #fff;
+  color: $buttonColor;
```

Comment:
Ensure that the variable $buttonColor is defined. If it is not defined, it will cause a compilation error.